### PR TITLE
Add a note about Mock classes not considered for security issues

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

File: contributing/code/security.rst [hunk 1]
```diff
@@ -21,6 +21,10 @@ email for confirmation):
   production (including the web profiler or anything enabled when ``APP_DEBUG``
   is set to ``true`` or ``APP_ENV`` set to anything but ``prod``);
 
+* Any security issues found in classes provided to help fo testing that should
```
Comment (derrabus) [inline contributing/code/security.rst:24]:
```suggestion
* Any security issues found in classes provided for testing that should
```

File: contributing/code/security.rst [hunk 2]
```diff
@@ -21,6 +21,10 @@ email for confirmation):
...
+  never be used in production (like for instance mock classes that contain
+  ``Mock`` in their name);
```
Comment (stof) [inline contributing/code/security.rst:26]: I suggest mentioning the `Test` namespace as well inside those braces.

**Comment by javiereguiluz** [comment]
Thanks Fabien!

File: contributing/code/security.rst (hunk 2 above)
Comment (javiereguiluz) [reply contributing/code/security.rst:26]: Added while merging. See https://github.com/symfony/symfony-docs/commit/ea432f9c2516b7c89662eaf6bf83ff6efaeafc81 Thanks.

File: contributing/code/security.rst (hunk 1 above)
Comment (javiereguiluz) [reply contributing/code/security.rst:24]: Fixed while merging. Thanks.

**Pull request merged**